rds_autostart.yml

```diff
@@ -231,7 +231,7 @@ Resources:
       Role: !GetAtt rdsAutoStopWorkerFunctionRole.Arn
       Timeout: 30
       Handler: index.handler
-      Runtime: python3.6
+      Runtime: python3.9
       Code:
         ZipFile: |
```

Comment:
code not updated per descriptions, deployment will include this old code.  Dont see logger changes

Reply:
Addressed